controllers/tools/visualization/visualizeSimulation_iDyntree.m

```diff
@@ -0,0 +1,71 @@
+function figureCont = visualizeSimulation_iDyntree(chi,CONFIG)
+%VISUALIZESIMULATION_IDYNTREE calls iDyntree visualizer to simulate robot
+%                             movements.
+%
+% figureCont = VISUALIZESIMULATION_IDYNTREE(chi,CONFIG) takes as input the
+% robot state, chi, and the configuration parameters.
+% The output is a counter for the automatic correction of figures numbers 
+% in case a new figure is added.
+%
+% Author : Gabriele Nava (gabriele.nava@iit.it)
+% Genova, January 2017
+%
+
+% ------------Initialization----------------
+%% initial parameters
+figureCont       = CONFIG.figureCont;
+mdlLdr           = CONFIG.mdlLdr;
+consideredJoints = CONFIG.consideredJoints;
+
+% get the joint position
+CONFIG.demux.baseOrientationType = 1;
+[~,qj,~,~] = stateDemux(chi',CONFIG);
+
+% visualization time
+init_time  = 1;
+end_time   = length(qj(1,:));
+
+% load the model from .urdf
+mdlLdr.loadReducedModelFromFile('model/model.urdf',consideredJoints);
```

Comment:
But for doing that we should first make sure that we install the meshes together with the urdf, something that at the moment we are not doing (see https://github.com/robotology-playground/icub-model-generator/issues/28 and https://github.com/robotology-playground/icub-model-generator/issues/33).